testing/networks/80084/config.toml

```diff
@@ -365,7 +365,7 @@ wal_dir = ""
 size = 0
 
 # Maximum size in bytes of a single transaction accepted into the mempool.
-max_tx_bytes = 1048576
+max_tx_bytes = 104857600 // 100 MB high enough to be impossible to hit
```

Comment:
@chuck-bear Do we run network out of this file?
If so, it may be enough for a quick patch to change this value in the config file.
If not, I will have to add a config flag to override the default config value.